content/docs/start/index.md

```diff
@@ -14,7 +14,7 @@ should be enough to understand the idea of DVC), or try to run them and get the
 fist hand experience.
 
 We'll be building an NLP project from scratch together. The end result is
-published on [Github](https://github.com/iterative/example-get-started).
+published on [GitHub](https://github.com/iterative/example-get-started).
```

Comment:
Thanks although this is not a typo. It's the formal way to write a proper name in English. We don't apply one or the other consistently at the moment, unfortunately, so this change can stay in for now.